README.md

```diff
@@ -11,23 +11,190 @@ easily in various projects.
 
 This repository is the Symfony bundle based on the [PHP library](https://github.com/acmephp/core).
 
+Installation
+------------
+
+The recommended way to install this bundle is through
+[Composer](http://getcomposer.org/). Require the `acmephp/symfony-bundle`
+package into your `composer.json` file:
+
+```json
+{
+    "require": {
+        "acmephp/symfony-bundle": "@stable"
+    }
+}
+```
+
+**Protip:** you should browse the
+[`acmephp/symfony-bundle`](https://packagist.org/packages/acmephp/symfony-bundle)
+page to choose a stable version to use, avoid the `@stable` meta constraint.
+
+Update `app/AppKernel.php`:
+
+```php
+public function registerBundles()
+{
+    $bundles = array(
+        // ...
+        new AcmePhp\Bundle\AcmePhpBundle(),
+    );
+
+    return $bundles;
+}
+```
+
+Configuration reference
+-----------------------
+
+```yml
+# app/config/config.yml
+
+acme_php:
+    # Certificates locations. Default: `~/.acmephp`
+    # Beware to use a directory readable by the web server
+#    certificate_dir: ~/.acmephp
```

Comment:
That's not what I meant, I'm just saying the directory may also need to be writable (in addition to `Beware to use a directory readable by the web server`).